core/src/main/java/com/kin/ecosystem/core/network/model/OutgoingTransferRequest.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Kin Ecosystem
+ * Apis for client to server interaction
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+package com.kin.ecosystem.core.network.model;
+
+import java.util.Objects;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * OutgoingTransferRequest
+ */
+public class OutgoingTransferRequest {
```

Comment:
I don't like the name as it not represent a request, only the payload.

Reply:
@OrenZak  what would you call it in your name spaces

Reply:
`OutgoingTransferInfo`, `OutgoingTransferData`, `OutgoingTransfer` ?